internal/twofa/enable2fa.go

```diff
@@ -48,12 +49,13 @@ func (c *Client) Enable() *cobra.Command {
 
 			cmd.SilenceUsage = true
 			if sms {
-				_, err := c.Service.EnableSms(token)
+				_, err := c.Service.EnableTfaSms(context.Background()).Execute()
```

Comment:
Do we need to be able to pass in the token?  Is this a spec issue that needs to be fixed?